DOCUMENTATION.md

```diff
@@ -0,0 +1,232 @@
+# Amazon Connect Chat Interface Documentation
+
+Looking to build a custom chat-interface for Amazon Connect Chat? Learn how to set up this package for local development, see overview of React components, and enable/configure existing features.
+
+> For additional common features (e.g. customer name from a form), check out the [customChatWidget](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/customChatWidget) example.
+
+## Table of contents
+
+- [Documentation](#documentation) - additional documentation reference
+- [Features](#features) - overview of global configuration options (logger, feature flags, etc)
+- [Customization](#customization)- high level overview of some of the major components
+- [Components](#customization) - details on how to enable/use various features
+- [Usage](#usage) - instructions to use `amazon-connect-chat-interface.js` file in an html page
```

Comment:
This isn't anchoring to anything for me.  Did we remove this section?

Reply:
Good catch, will remove in next revision